examples/complete/variables.tf

```diff
@@ -1,3 +1,9 @@
 variable "region" {
   type = string
 }
+
+variable "allow_ssl_requests_only" {
+  type        = bool
+  default     = false
+  description = "Set to `true` to require requests to use Secure Socket Layer (HTTPS/SSL). This will explicitly deny access to HTTP requests"
```

Comment:
We should probably default this to `true` or it will be flagged by Security Hub (I know this is only an example)

Reply:
Makes sense, done!